src/main/java/serverutils/handlers/ServerUtilitiesClientEventHandler.java

```diff
@@ -282,6 +285,19 @@ public void onClientTick(TickEvent.ClientTickEvent event) {
         }
     }
 
+    @SubscribeEvent
+    public void onGuiScreenDraw(final GuiScreenEvent.DrawScreenEvent.Post event) {
+        if (ClientUtils.areButtonsVisible(event.gui)) {
+            event.gui.buttonList.forEach((GuiButton button) -> {
+                if (button instanceof GuiSidebar sidebar) {
+                    sidebarButtonTooltip.clear();
+                    sidebar.addTooltip(sidebarButtonTooltip);
+                    event.gui.func_146283_a(sidebarButtonTooltip, event.mouseX, event.mouseY);
```

Comment:
On second thought using the event isn't necessary, this can just be moved to where the old tooltip rendering code was.

Reply:
I can't leave the rendering code in the old location, because it's run too early and various elements will draw over it, like DuraDisplay, TinCon inventory tabs, various labels, and so on as mentioned in the linked GTNH issue and shown in the screenshots above.

Reply:
Yeah alright that makes sense